Pmods/PmodCDC1/AD7156.c

```diff
@@ -173,12 +173,14 @@ void AD7156_WriteReg(int reg, unsigned char txData)
 void AD7156_BurstWriteReg(int reg, char size, int txData)
 {
 	unsigned char txBuffer[4] = {0x00, 0x00, 0x00, 0x00};
+	int	i = 0;
```

Comment:
this is just a style thing, feel free to disregard;
but you don't need the space between `int` & `i`

and you also don't need to initialize `i` here, since it gets initialized in the for loop